specification/2.0/appendix/performance-metrics/README.md

```diff
@@ -1,6 +1,6 @@
 
 --- Name -----------Property    ----- Description ----------------------------
-[verticecount]      Scene       Total number of vertices used by a model in a scene
+[vertexcount]       Scene       Total number of vertices used by a model in a scene
```

Comment:
I believe this is intended to be the total number of vertices used by **all models** in a scene.
But it says `used by a model`.  Potentially confusing.

Reply:
The glTF asset can contain one or more scenes.
Each scene is a Node hierarchy.
I agree that model is ambigous because there is no json property with that name.
I would say that a glTF asset is usually called the model.
In this scenario I think it could be better to specify:
> Total number of vertices used by the Nodes in a scene.